src/main/java/org/truffleruby/core/fiber/FiberManager.java

```diff
@@ -79,7 +79,14 @@ public String getFiberDebugInfo() {
             builder.append("  fiber @");
             builder.append(ObjectIDOperations.verySlowGetObjectID(context, fiber));
             builder.append(" #");
-            builder.append(Layouts.FIBER.getThread(fiber).getId());
+
+            final Thread thread = Layouts.FIBER.getThread(fiber);
+
+            if (thread == null) {
+                builder.append("(no Java thread)");
```

Comment:
How can this happen? Aren't fibers always attached to a Ruby thread, which is always attached to a Java thread?

Reply:
It's cleared during shutdown.